dashboards-observability/public/components/custom_panels/__tests__/__snapshots__/custom_panel_table.test.tsx.snap

```diff
@@ -1926,7 +1926,7 @@ exports[`Panels Table Component renders panel table container 1`] = `
                                           <div
                                             className="euiTableCellContent euiTableCellContent--overflowingContent"
                                           >
-                                            11/02/2021 07:26 PM
+                                            11/03/2021 12:56 AM
```

Comment:
same here and follow the approach we ae following for other PRs

Reply:
Done